module/account/src/Infrastructure/Handler/CreateUserCommandHandler.php

```diff
@@ -44,21 +45,26 @@ public function __invoke(CreateUserCommand $command): void
         foreach ($activeLanguages as $activeLanguage) {
             $languagePrivilegesCollection[$activeLanguage->getCode()] = new LanguagePrivileges(true, true);
         }
-        $user = new User(
+
+        $helpUser = $this->createUser($command, $command->getPassword(), $languagePrivilegesCollection);
```

Comment:
```suggestion
        $plainPasswordUser = $this->createUser($command, $command->getPassword(), $languagePrivilegesCollection);
```
perhaps? Help does not describe what it actually does